gcloud-java-compute/src/main/java/com/google/gcloud/compute/ImageInfo.java

```diff
@@ -393,7 +393,7 @@ Image toPb() {
   /**
    * Returns a builder for an {@code ImageInfo} object given the image identity and an image
    * configuration. Use {@link DiskImageConfiguration} to create an image from an existing disk. Use
-   * {@link RawImageConfiguration} to create an image from a raw image stored in Google Cloud
+   * {@link StorageImageConfiguration} to create an image from a raw image stored in Google Cloud
```

Comment:
I agree this might be confusing. I am updating the javadoc here and elsewhere to mention the Storage file, rather than the "raw" concept.